### ISSUE TITLE
vim.schedule error

### ISSUE THREAD

**Issue by maxthier** [topaxi/pipeline.nvim#20]
### Error

When using this plugin i receive the following error all the time:
```
Error executing vim.schedule lua callback: /usr/share/nvim/runtime/lua/vim/iter.lua:236: flatten() requires a list-like table
stack traceback:                                                                                                                                                                                                   
        [C]: in function 'error'                                                                                                                                                                                   
        /usr/share/nvim/runtime/lua/vim/iter.lua:236: in function 'flatten'                                                                                                                                        
        ...ipeline.nvim/lua/pipeline/providers/github/rest/init.lua:76: in function ''                                                                                                                             
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

### When does it happen?

- Open a GitHub project with actions
- <leader>ci
- error starts appearing all the time

As soon as I run `:Pipeline close` it doesn't happen anymore.

I don't now much about Lua and neovim plugins in general but if you have anything I can help with I'm happy to

**Comment by topaxi** [comment]
Could you try again with the latest version on `main`?

Unsure whether it's actually fixed, but [e2a2da5](https://github.com/topaxi/pipeline.nvim/commit/e2a2da566cd7e745a3b8097567e0367bb5eac93e) might do.

**Comment by topaxi** [comment]
Feel free to reopen if the problem persists and I'll take another look :)
